src/Nox.Types.EntityFramework/Types/Year/YearConverter.cs

```diff
@@ -4,5 +4,5 @@ namespace Nox.Types.EntityFramework.Types;
 
 public class YearToUShortConverter : ValueConverter<Year, ushort>
 {
-    public YearToUShortConverter() : base(year => year.Value, n => Year.From(n)) { }
+    public YearToUShortConverter() : base(year => year.Value, n => Year.FromDatabase(n)) { }
```

Comment:
Can we rename the variable "n" to "yearValue", to keep the consistency?

Reply:
Done